ios/FluentUI/Table View/TableViewTokens.swift

```diff
@@ -4,172 +4,329 @@
+        case subtitleThreeLineHeight
+
+        /// The minimum height for the footer label.
+        case footerHeight
+
+        /// The leading margin for the labelAccessoryView.
+        case labelAccessoryViewMarginLeading
+
+        /// The trailing margin for the labelAccessoryView.
+        case labelAccessoryViewMarginTrailing
+
+        /// The leading margin for the customAccessoryView.
+        case customAccessoryViewMarginLeading
+
+        /// The minimum vertical margin for the customAccessoryView.
+        case customAccessoryViewMinVerticalMargin
+
+        /// The vertical margin for the label when it has one or three lines.
+        case labelVerticalMarginForOneAndThreeLines
+
+        /// The vertical margin for the label when it has two lines.
+        case labelVerticalMarginForTwoLines
+
+        /// The vertical spacing for the label.
+        case labelVerticalSpacing
+
+        /// The minimum TableViewCell height; the height of a TableViewCell with one line of text.
+        case minHeight
+
+        /// The height of a TableViewCell with two lines of text.
+        case mediumHeight
+
+        /// The height of a TableViewCell with three lines of text.
+        case largeHeight
 
-    /// The grouped background color of the TableViewCell.
-    open var cellBackgroundGrouped: DynamicColor {
-        .init(light: aliasTokens.backgroundColors[.neutral1].light,
-              dark: aliasTokens.backgroundColors[.neutral3].dark,
-              darkElevated: ColorValue(0x212121))
+        /// The trailing margin for the selectionImage.
+        case selectionImageMarginTrailing
+
+        /// The size for the selectionImage.
+        case selectionImageSize
+
+        /// The duration for the selectionModeAnimation.
+        case selectionModeAnimationDuration
+
+        /// The minimum width for any text area.
+        case textAreaMinWidth
+
+        /// The alpha value that enables the user's ability to interact with a cell.
+        case enabledAlpha
+
+        /// The alpha value that disables the user's ability to interact with a cell; dims cell's contents.
+        case disabledAlpha
+
+        /// The default horizontal spacing in the cell.
+        case horizontalSpacing
+
+        /// The leading padding in the cell.
+        case paddingLeading
+
+        /// The vertical padding in the cell.
+        case paddingVertical
+
+        /// The trailing padding in the cell.
+        case paddingTrailing
+
+        /// The color for the accessoryDisclosureIndicator.
+        case accessoryDisclosureIndicatorColor
+
+        /// The color for the accessoryDetailButtonColor.
+        case accessoryDetailButtonColor
+
+        /// The main primary brand color of the theme.
+        case mainBrandColor
+
+        /// The destructive text color in an ActionsCell.
+        case destructiveTextColor
+
+        /// The communication text color in an ActionsCell.
+        case communicationTextColor
```

Comment:
I don't think `destructiveTextColor` and `communicationTextColor` were intended to be part of the main TableViewCell token class. Since only `ActionCell` uses them.

Reply:
That's correct, and this is something I would like us to revisit in the future. In the current new system, only the base token set can define the set of token values in the `Tokens` enum. Swift doesn't allow extending enums for pretty good reasons, but that means children can't easily add new values.

There are ways around this, including containerization, but in the name of expediency I'm taking the easiest solution _for now_, with an intent to revisit in the future.